### PR TITLE
libplatsupport: use KernelPlatformZynqmp

### PR DESCRIPTION
**Pull request by axel-h**
Follow-up from https://github.com/seL4/util_libs/pull/142#discussion_r1013337980

### CODE REVIEW

File: libplatsupport/CMakeLists.txt [hunk 1]
```diff
@@ -134,7 +134,7 @@ if(KernelPlatImx8mq OR KernelPlatformImx8mm-evk)
     target_include_directories(platsupport PUBLIC plat_include/imx8m)
 endif()
 
-if(KernelPlatformZynqmpZcu102 OR KernelPlatformZynqmpUltra96 OR KernelPlatformZynqmpUltra96v2)
+if(KernelPlatformZynqmp)
     # This still needs to be cleaned up, so there is a mach folder "zynymq" and
```
Comment (chrisguikema) [inline libplatsupport/CMakeLists.txt:138]: I know this isn't part of the PR, but "zynqmp" is mangled here. Can you fix?

Comment (axel-h) [reply libplatsupport/CMakeLists.txt]: I've added a commit that fixed the comment

**Comment by chrisguikema** [comment]
Looks good to me!

**Comment by lsf37** [comment]
Looks good, just checking that the tests work before I merge.

**Comment by axel-h** [comment]
Can we merge this now?

**Comment by lsf37** [comment]
Restarted the test to check.

**Comment by axel-h** [comment]
It seems there is a general problem with starting the board now? Building works, but the image can't be loaded:
```
 '/tmp/tmp.PTTuIEl9bv' -> '/tftpboot/zcu102/sel4-image'
  Couldn't change mode of /tftpboot/zcu102/sel4-image: [Errno 1] Operation not permitted: '/tftpboot/zcu102/sel4-image'
```

**Comment by lsf37** [comment]
> It seems there is a general problem with starting the board now? Building works, but the image can't be loaded:
> 
> ```
>  '/tmp/tmp.PTTuIEl9bv' -> '/tftpboot/zcu102/sel4-image'
>   Couldn't change mode of /tftpboot/zcu102/sel4-image: [Errno 1] Operation not permitted: '/tftpboot/zcu102/sel4-image'
> ```

The mode change error is normal (all logs have it, we should really remove that command), but the subsequent boot failure looks like the board is not reachable, yes.

**Comment by axel-h** [comment]
Not sure why `IMX8MM_EVK_debug_gcc_32` failed in the middle, ist is a power spike that cause a reset?
```
Thu, 27 Apr 2023 15:05:35 GMT  Running test SERSERV_PARENT_010 (Test a series of unexpected input values to write())
Thu, 27 Apr 2023 15:05:35 GMT  serial_server_write@clientapi.c:221 Serserv Client: printf: NULL passed for required arguments.
Thu, 27 Apr 2023 15:05:35 GMT  	Is connection handle valid?
Thu, 27 Apr 2023 15:05:35 GMT  serial_server_write@clientapi.c:221 Serserv Client: printf: NULL passed for required arguments.
Thu, 27 Apr 2023 15:05:35 GMT  	Is connection handle valid?
Thu, 27 Apr 2023 15:05:35 GMT  Test SERSERV_PARENT_010 passed
Thu, 27 Apr 2023 15:05:36 GMT  
Thu, 27 Apr 2023 15:05:36 GMT  U-Boot SPL 2018.03 (Dec 19 2018 - 17:39:10 +0800)
Thu, 27 Apr 2023 15:05:36 GMT  power_bd71837_init
```
Next test should have been `SYNC001`

**Comment by lsf37** [comment]
> Not sure why `IMX8MM_EVK_debug_gcc_32` failed in the middle, is it a power spike that cause a reset?

@wom-bat has been trying to figure this out for months now, it's just frequent enough to fail in every other larger test, but too infrequent to reproduce manually. We thought it was temperature, but that does seem to be controlled now. It only seems to be happening when load is high on multiple boards, so maybe it is something with power (when you re-run the one board in isolation it almost always passes everything, i.e. it's not completely gone, but frequency of failure drops strongly). Extremely annoying to debug.

Might just be that some component on the board is old and failing. We have new `imx8mm` boards on order (also just to have more of them in CI), but delivery time is still a few months, I think.

In any case, it's not related to this PR and Ok to ignore the failure for this.

**Pull request merged**